howto/manage/web-dashboard.md

```diff
@@ -74,3 +74,37 @@ You can select any application you previously created and start a streaming sess
 ![Start a streaming session|690x352](https://assets.ubuntu.com/v1/6087fbd9-gs_dashboard_start_session.png)
 
 For more information about the Streaming Stack, visit [About application streaming](https://discourse.ubuntu.com/t/streaming-android-applications/17769).
+
+#### Streaming statistics
+
+View the streaming statistics for your running sessions by selecting the **Statistics** button on the session. The statistics display on the right pane and also have a download option to download the statistics in a `.csv` format for further analysis.
+
+The downloaded `.csv` file has the following statistics:
+
+| Statistic | Description |
+| --------- |------------ |
+| `date` | Date and time in ISO 8601 format |
+| `network-currentrtt` | Current round-trip time of the network |
+| `video-bandwidth` | The amount of video data that the session can handle per second |
+| `video-totalreceived` | Total video data received |
```

Comment:
To be accurate, Is it the total number of video data received or the total video data received in bytes?

Reply:
The csv file (see Michele's upload in a comment above) shows that the data is in MB, so "total" seems like the correct representation than "total number"

Reply:
Okay, If the unit type is steady and won't change during the runtime E.g. MB - KB. I feel like we want to be more explicit on the doc.
What do you think?

Reply:
Even if it changes in runtime, we are talking about downloading the statistics that are displayed on the statistics panel. At the time of download, it will be more absolute and has the corresponding unit in the CSV file. So I think we can generalise here with just "total".

Reply:
Okay